tests/test_caching.py

```diff
@@ -248,7 +248,7 @@ def test_cache_delete_single(self):
         trace_dir = os.path.dirname(trace_path)
         trace_file = os.path.basename(trace_path)
         cache_dir = '.' + trace_file + '.cache'
-        number_of_trace_categories = 32
+        number_of_trace_categories = 31
```

Comment:
this is weird.  You are not changing the inputs to this test, so why did you have to reduce the number of trace categories?

Reply:
From what I could see, there was one CSV file per event class name. Since I deleted the "Print" class, this one disappeared.

Reply:
ah, ok.  What confused me is that it was in the commit "event: rename thermal_power_cpu_get" instead of "events: remove print event".  Makes sense now.